dask/tests/test_sizeof.py

```diff
@@ -37,10 +37,12 @@ def test_pandas():
     assert sizeof(df.x) >= sizeof(df.index)
     assert sizeof(df.y) >= 100 * 3
     assert sizeof(df.index) >= 20
+    assert sizeof(df.set_index(['x', 'y'].index)) < 1800
```

Comment:
It looks like this is causing a FutureWarning in Pandas.  I'm not sure exactly why.  (this is reported in the CI)

Reply:
Whoops just a typo, should be fixed now